backend/development.ini.sample

```diff
@@ -383,6 +383,26 @@ caldav.radicale.headers.Access-Control-Allow-Methods =  GET, POST, OPTIONS, PROP
 caldav.radicale.headers.Access-Control-Allow-Headers =  User-Agent, Authorization, Content-type, Depth, If-match, If-None-Match, Lock-Token, Timeout, Destination, Overwrite, X-client, X-Requested-With, Prefer
 caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 
+####
+# SEARCH (ElasticSearch)
+####
+search.enabled = False
```

Comment:
@lebouquetin @basilelegal @inkhey Do we should rename these parameters ? `search` is in reality an advanced search (standard search is enabled by default).

Reply:
I think search need to be all the time activated, so We just need to choice if its "simple" or "elasticsearch"

Reply:
Like we have for "auth_types = "

Reply:
@buxx search is search. There is no such thing as advanced search or simple search in actual code, there is search and multiple engine for it.

Reply:
@inkhey the option search.enabled is useless because we'll always have a search available (there are no reason for disabling the search feature)